.github/workflows/ubuntu_build.yml

```diff
@@ -14,26 +14,30 @@ on:
 
 permissions:
   contents: read
-env: 
-    DPDK_VERSION: 25.03
+
 jobs:
   changes:
     runs-on: ubuntu-latest
     permissions:
       pull-requests: read
     outputs:
-      changed: ${{ steps.filter.outputs.ubuntu_build == 'true' }}
+      base: ${{ steps.filter.outputs.ubuntu_build == 'true' }}
+      afxdp_build: ${{ steps.filter.outputs.afxdp_build == 'true' }}
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      
       - uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v2
         id: filter
         with:
           filters: .github/path_filters.yml
+    env:
+      SETUP_ENVIRONMENT: 0
+      BUILD_AND_INSTALL_EBPF_XDP: 0
+      BUILD_AND_INSTALL_DPDK: 0
+      BUILD_ICE_DRIVER: 0 # Only for CICD action builds
+      BUILD_AND_INSTALL_ICE_DRIVER: 0
```

Comment:
this refers to `changes:`, so for readability should be before steps (steps 99% of time should be the last in current `yaml` tree.